docs/src/index.ejs

```diff
@@ -32,24 +31,14 @@
   </style>
 </head>
 <body>
-  <script src="https://cdn.jsdelivr.net/npm/@babel/polyfill@7/dist/polyfill.min.js"></script>
+  <script src='https://cdn.jsdelivr.net/npm/core-js-bundle/minified.js'></script>
```

Comment:
`core-js-bundle` is bundled version of `core-js` which is a successor for `@babel/polyfill`:
https://babeljs.io/docs/en/babel-polyfill